ui/src/pages/QuickSetup/ProvideServicesPage.tsx

```diff
@@ -458,57 +375,24 @@ const CreateServiceRequests = (props: {
               break;
             case ServiceKind.CLEARING:
               const clearingAccounts = requestInfo?.accounts;
-              if (!clearingAccounts?.clearingAccount || !clearingAccounts?.marginAccount) {
+              if (!clearingAccounts?.clearingAccount) {
                 return;
               }
               const clearingParams = {
                 ...params,
                 clearingAccount: clearingAccounts.clearingAccount,
-                marginAccount: clearingAccounts.marginAccount,
               };
               await ledger.create(ClearingRequest, clearingParams);
```

Comment:
this can be a `doRequest` right?

Reply:
Only would prefer that if a `displayErrorMessage` could also be added to that function, uniformly handling failed requests for all

Reply:
> this can be a `doRequest` right?

`ClearingAccount` is required for the clearing service which is why `doRequest` won't work in its current form.

> Only would prefer that if a `displayErrorMessage` could also be added to that function, uniformly handling failed requests for all

I agree but I think this should be part of a separate PR.